Makefile

```diff
@@ -103,8 +102,13 @@ delete: ## Delete the controller from your ~/.kube/config cluster
 	helm uninstall karpenter --namespace karpenter
 
 codegen: ## Generate code.
-	curl https://raw.githubusercontent.com/aws/karpenter-core/v0.0.1/chart/crds/karpenter.sh_provisioners.yaml > charts/karpenter/crds/karpenter.sh_provisioners.yaml
+	controller-gen \
```

Comment:
Somehow this got lost from karpenter's release processes 🤔

Reply:
Good catch!